scripts/core/editor2/add-embed.ctrl.js

```diff
@@ -1,6 +1,16 @@
 var embedCodeHandlers = require('./embedCodeHandlers.js');
 
-angular.module('superdesk.apps.editor2.embed', []).controller('SdAddEmbedController', SdAddEmbedController);
+angular.module('superdesk.apps.editor2.embed', []).controller('SdAddEmbedController', SdAddEmbedController)
+    .config(['$sceDelegateProvider', ($sceDelegateProvider) => {
+        $sceDelegateProvider.resourceUrlWhitelist([
```

Comment:
this list repeats 2 times in https://github.com/superdesk/angular-embed/pull/5/files and here ones

maybe it should be in variable, what do you think?